en/docs/guides/login/oidc-discovery.md

```diff
@@ -1,16 +1,16 @@
-## Discover OpenID Connect Provider
+## Discover OpenID Connect provider
 
 This page guides you through using [OpenID Connect Discovery](../../../concepts/authentication/discovery) to discover an end user's OpenID provider, and to obtain information required to interact with the OpenID provider, including its OAuth 2.0 endpoint locations. 
 
-The OpenID Connect discovery endpoint is as follows:
+You can use this OIDC Discovery document when you need applications to be automatically configured. The OpenID Connect discovery endpoint is as follows:
```

Comment:
You can use this OIDC Discovery document to automatically configure applications.